<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>654</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_asset_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>자산 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view_assets_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>자산 보기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_cvt_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CVT 계산</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
